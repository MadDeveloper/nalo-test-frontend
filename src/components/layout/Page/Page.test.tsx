import { BrowserRouter } from "react-router-dom"
import renderer from "react-test-renderer"
import { Page } from "."

describe("Components > Page", () => {
  it("should match snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Page />
        </BrowserRouter>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
