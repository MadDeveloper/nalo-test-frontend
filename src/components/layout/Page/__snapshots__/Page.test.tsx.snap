// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Page should match snapshot 1`] = `
<main
  className="root"
  style={
    Object {
      "opacity": 0,
    }
  }
/>
`;
